### PR TITLE
#2038 - updated config and partition table templates using ones from the community repository

### PR DESCRIPTION
**Pull request by dmitri-d**

### CODE REVIEW

**Comment by ohadlevy** [comment]
@witlessbird would you update the tests as well? - thanks

**Comment by dmitri-d** [comment]
@ohadlevy, @domcleal: tests are green (pls. see comments in tests though).

File: app/views/unattended/preseed.erb [hunk 1]
```diff
@@ -46,11 +52,26 @@ user-setup-udeb passwd/root-login boolean true
 d-i passwd passwd/make-user boolean false
 user-setup-udeb passwd/make-user boolean false
 
+<% if @host.params['enable-puppetlabs-rep'] && @host.params['enable-puppetlabs-rep'] == 'true' -%>
```
Comment (GregSutcliffe) [inline app/views/unattended/preseed.erb]: This was fixed in the community repo i thought - should be "repo" not "rep"

File: db/migrate/20090730152224_create_ptables.rb [hunk 2]
```diff
@@ -7,8 +7,28 @@ def self.up
       t.references :operatingsystem
       t.timestamps
     end
-    Ptable.create :name => "RedHat default", :layout =>"zerombr yes\nclearpart --all --initlabel\npart /boot --fstype ext3 --size=100 --asprimary\npart /     --fstype ext3 --size=1024 --grow\npart swap  --recommended"
-    Ptable.create :name => "Ubuntu default", :layout =>"d-i partman-auto/disk string /dev/sda\nd-i partman-auto/method string regular\nd-i partman-auto/init_automatically_partition select Guided - use entire disk\nd-i partman/confirm_write_new_label boolean true\nd-i partman/choose_partition select \\\nFinish partitioning and write changes to disk\nd-i partman/confirm boolean true\n"
+    Ptable.create :name => "RedHat default", :layout =>"zerombr yes\nclearpart --all --initlabel\nautopart"
```
Comment (domcleal) [inline db/migrate/20090730152224_create_ptables.rb]: Should just be `zerombr`, maybe add a trailing newline as well to be safe?

File: app/views/unattended/preseed.erb (hunk 1 above)
Comment (domcleal) [inline app/views/unattended/preseed.erb]: It wasn't, I've just pushed https://github.com/theforeman/community-templates/commit/5d612f1d7eecf4e1961fa977a60bd04540ff1b5f, so can you fix the same here please @witlessbird?

**Comment by domcleal** [comment]
:+1: tested and verified against community-templates.  Two small fixes before merge, see comments above.

**Comment by dmitri-d** [comment]
@domcleal: ping?

File: bundler.d/libvirt.rb [hunk 3]
```diff
@@ -1,3 +1,3 @@
 group :libvirt do
-  gem "ruby-libvirt", :require => 'libvirt'
+#  gem "ruby-libvirt", :require => 'libvirt'
```
Comment (domcleal) [inline bundler.d/libvirt.rb:2]: _cough_

`bundle install --without libvirt` :D

**Comment by domcleal** [comment]
Merged as cb92c0b, but with libvirt support added back :)  Thanks @witlessbird!

**Comment by dmitri-d** [comment]
@domcleal: thwarted. again.

**Pull request closed without merging**